zetaclient/common/constant.go

```diff
@@ -1,6 +1,11 @@
 package common
 
+import "time"
+
 const (
+	// ZetaBlockTime is the block time of the ZetaChain
+	ZetaBlockTime = 6000 * time.Millisecond
```

Comment:
Is the value needs to be accurate for the watch process? Because it can vary, often less than 6

Reply:
It's used by outbound tracker reporter and the orchestrator to monitor zetacore changes. It doesn't need accuracy ATM.